practice-app/website/test/event_test.py

```diff
@@ -1,26 +1,117 @@
 import unittest
-import requests
-from ..api import event
-from ...main import app
-
-class TestEventAPI(unittest.TestCase):
-
-    def setUp(self):
-        app.testing = True
-        self.app = app.test_client()
-
-    def test_events_POST(self):
-
-        # Foreign key error
-        # Adding without an error
-        # without required fields
-        # 
-        response = self.app.post('/api/v1.0/events', json={
-            'name': '', 
-            'date': '',
-            'location': '',
-            'creator_user': '',
-        })
-        json_data = response.get_json()
-        self.assertEqual(json_data.status, '200 OK')
+import requests, json
 
+
+# Checks the number of messages sent in the discussion
+def hasMoreThan2Messages(messages):
+    return len(messages) > 2
+
+class hasMoreThan2MessagesTest(unittest.TestCase):
```

Comment:
Maybe to have consistent naming, starting the test class name with Test would be better

Reply:
our common approach is the other way, please respect us

Reply:
I have made corresponding changes, so I will send it when you send too in the desired format.